packages/libgconf.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Libgconf < Package
+  description 'GConf is a system for storing application preferences.'
+  homepage 'https://projects-old.gnome.org/gconf/'
+  version '3.2.6'
+  source_url 'https://ftp.gnome.org/pub/GNOME/sources/GConf/3.2/GConf-3.2.6.tar.xz'
+  source_sha256 '1912b91803ab09a5eed34d364bf09fe3a2a9c96751fde03a4e0cfa51a04d784c'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'dbus_glib'
+  depends_on 'gtk3'
+  depends_on 'gtk2'
```

Comment:
This dependency is not needed here.  The gtk2 dependency is needed for code, not libgconf.

Reply:
I think libgconf has some gtk2 bindings, too. I looked at the configure log.

Reply:
It looks for gtk3, not gtk2.

Reply:
OK. Will change.